mousestyles/dynamics/tests/test_dynamics.py

```diff
@@ -98,49 +98,95 @@ def test_get_prob_matrix_small_interval():
     assert example[0, 2] == 0.2
     assert example[1, 2] == 1.
     assert sum(example[0, :]) == 1.
-
```

Comment:
I think it is because weiyan import the function get_score in the same line of importing mcmc_simulation since they are in a same file, say __init__.py. @harryh1993